src/toil_scripts/gatk_processing/gatk_preprocessing.py

```diff
@@ -615,11 +444,12 @@ def generate_config():
         # Edit the values in this configuration file and then rerun the pipeline
         # Comments (beginning with #) do not need to be removed. Optional parameters may be left blank.
         ##############################################################################################################
-        ref:                      # Required: Reference Genome URL 
+        genome:                   # Required: Reference Genome URL
         phase:                    # Required: URL (1000G_phase1.indels.hg19.sites.fixed.vcf)
         mills:                    # Required: URL (Mills_and_1000G_gold_standard.indels.hg19.sites.vcf)
         dbsnp:                    # Required: URL (dbsnp_132_b37.leftAligned.vcf URL)
         ssec:                     # Optional: (string) Path to Key File for SSE-C Encryption
+        mock:                     # Optional: (bool) If True, run in mock mode
```

Comment:
Let me check this against the other pipelines, but IIRC, mock should be a cmd line parameter, not a config file setting.